tests/virt_autotest/prepare_non_transactional_server.pm

```diff
@@ -34,6 +35,7 @@ sub run {
     $self->prepare_services;
     $self->prepare_reboot;
     $self->restore_ground;
+    virt_autotest::virtual_network_utils::create_host_bridge_nm;
```

Comment:
Add a new subroutine prepare_network like existing prepare_xxx and put   virt_autotest::virtual_network_utils::create_host_bridge_nm in it.

Reply:
DONE

Reply:
I think it would be better to name it ```prepare_network```

Reply:
DONE